utils/holly/build-pr.jenkins

```diff
@@ -43,7 +43,7 @@ pipeline {
                 script {
                     // required configurations
                     def configurations = [
-                        [preset: "mini", build_type: "release", bootloader: "yes"],
+                        [preset: "mini", build_type: "release", bootloader: "yes",devel_items: "no"],
```

Comment:
build "yes" variant normally, "no" for release branch only

Reply:
👌🏻